crap-advisor-frontend/src/core/pages/MainPage/index.jsx

```diff
@@ -1,12 +1,20 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {Map, Sidebar} from "../../components";
 import "./index.css"
 
 const MainPage = () => {
+    const [selectedRestroom, setSelectedRestroom] = useState(null)
+
+    const selectRestroom = (restroomToSelect) => {
+        if (selectedRestroom === restroomToSelect) return
```

Comment:
Do not use empty return value in if condition. Better try ` if (selectedRestroom !== restroomToSelect) {your_code}` with no else. It's will be work fine.

Reply:
My bad